CypressTests/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "dansacademisationapicypresstests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```

Comment:
Worth updating the scripts section to include options for running Cypress tests.
e.g. see how [a2b internal](https://github.com/DFE-Digital/a2b-internal/blob/main/ApplyToBecomeInternal/ApplyToBecomeCypressTests/package.json#L7) does it